src/lib/viewers/doc/DocControls.tsx

```diff
@@ -7,23 +7,28 @@ import FullscreenToggle, { Props as FullscreenToggleProps } from '../controls/fu
 import PageControls, { Props as PageControlsProps } from '../controls/page';
 import ThumbnailsToggle, { Props as ThumbnailsToggleProps } from '../controls/sidebar';
 import ZoomControls, { Props as ZoomControlsProps } from '../controls/zoom';
+import { AnnotationColor } from '../../AnnotationModule';
+import { AnnotationMode } from '../controls/annotations/types';
 
 export type Props = AnnotationsControlsProps &
     ColorPickerControlProps &
     FindBarToggleProps &
     FullscreenToggleProps &
     PageControlsProps &
     ThumbnailsToggleProps &
-    ZoomControlsProps;
+    ZoomControlsProps & {
+        onAnnotationColorChange: (color: string) => void;
```

Comment:
This isn't consistent with the above props imports because `onAnnotationColorChange` is not the name that's being used in the ColorPickerControl props (it's using `onSelect` instead).